srcCpp03/PerftestTransport.cxx

```diff
@@ -505,7 +726,7 @@ bool configureTransport(
              */
             TransportBuiltinMask mask = qos.policy<TransportBuiltin>().mask();
```

Comment:
Why we need the mask on a variable?

Reply:
It is not a must, but it makes understanding the code simpler: that way we don't need to call the `qos.policy<TransportBuiltin>().mask()` function every time.